### PR TITLE
feat: temporary profile edit

### PR DESCRIPTION
**Pull request by JohnXander**

### CODE REVIEW

**Review by sirpae — APPROVED** [review]

File: src/components/profile/Profile.js [hunk 1]
```diff
@@ -1,27 +1,27 @@
-import { useState } from 'react';
 import Avatar from '@mui/material/Avatar';
 import Link from '@mui/material/Link';
 import './style.css'
 
-import Header from '../Header/Header';
 import EditForm from './EditForm';
+// import client from "../../utils/client";
```
Comment (Auenc) [inline src/components/profile/Profile.js:6]: Let's remove this commented line

File: src/components/profile/Profile.js [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
 
-        fetch(`${process.env.REACT_APP_API_URL}/user/${userId}`, {
-            method: "PATCH",
-            headers: { Authorization: token }
-        })
-            .then(resp => resp.json())
-            .then(data => setUser(data.data.user))
+        alert("Backend work in progress")
+
+        // client
```
Comment (Auenc) [inline src/components/profile/Profile.js:21]: Let's remove this commented code

File: src/components/profile/Profile.js [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
+            return
+        }
 
-        fetch(`${process.env.REACT_APP_API_URL}/user/${userId}`, {
-            method: "PATCH",
-            headers: { Authorization: token }
-        })
-            .then(resp => resp.json())
-            .then(data => setUser(data.data.user))
+        alert("Backend work in progress")
```
Comment (Auenc) [inline src/components/profile/Profile.js:19]: Let's not use alerts, as some browsers can block them.

**Review by Auenc — CHANGES_REQUESTED** [review]
Minor changes mentioned in the comments

**Review by Auenc — APPROVED** [review]

**Review by sirpae — APPROVED** [review]

**Pull request merged**